src/ontology/mondo-edit.obo

```diff
@@ -586149,6 +586152,15 @@ property_value: http://purl.org/dc/elements/1.1/date "2024-07-23T20:59:58Z" xsd:
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-0587-4693
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7831" xsd:anyURI
 
+[Term]
+id: MONDO:1040022
+name: linkeropathy
+def: "Group of rare heritable connective tissue disorders, characterized by a variable degree of short stature, skeletal dysplasia, joint laxity, cutaneous anomalies, dysmorphism, heart malformation, and developmental delay. The LK genes encode for enzymes that add glycosaminoglycan chains onto proteoglycans via a common tetrasaccharide linker region." [https://orcid.org/0000-0001-5493-2602, https://orcid.org/0000-0002-0587-4693, PMID:31196143, PMID:31438591, PMID:37657630]
+is_a: MONDO:0002254 ! syndromic disease
+is_a: MONDO:0005516 ! osteochondrodysplasia
```

Comment:
It looks like "connective tissue" should also be a parent term based on the definition.

Reply:
Addressed!